modules/permutiveRtdProvider.js

```diff
@@ -174,6 +159,21 @@ function updateOrtbConfig (currConfig, segmentIDs, sspSegmentIDs, transformation
   const updatedUserKeywords = (currentUserKeywords === '') ? keywords : `${currentUserKeywords},${keywords}`
   deepSetValue(ortbConfig, 'ortb2.user.keywords', updatedUserKeywords)
 
+  // Set bidder specific extensions
+  if (bidder === 'rubicon') {
+    if (segmentIDs.length > 0) {
+      deepSetValue(ortbConfig, 'ortb2.user.ext.data.' + PERMUTIVE_STANDARD_KEYWORD, segmentIDs)
+    }
+
+    if (segmentData?.rubicon?.length > 0) {
```

Comment:
same question - why isn't this such that each `bidder` picks `segmentData[bidder]`?

Reply:
I think the [above comment should answer this](https://github.com/prebid/Prebid.js/pull/9555#discussion_r1129223057), it's a specific extension for Rubicon.

Reply:
This is a rules violation; you are not allowed to have a "specific extension for Rubicon."